src/index.ts

```diff
@@ -467,9 +479,13 @@ export async function getJsonStructure(accessToken: string, config: Configuratio
 
 //Main function
 async function main() {
-	const accessToken = getAccessToken()
-	const config = loadConfig()
-	writeFile("cachedData.json", await getJsonStructure(accessToken, config));
+	try{
+		const accessToken = getAccessToken()
+		const config = loadConfig()
+		writeFile("cachedData.json", await getJsonStructure(accessToken, config));
+	} catch{
+		writeFile("cachedData.json", "{\"aux\":{ \"crawlStart\": \"" + Date.now().toString() + "\", \"error\": \"" + error.msg + "\"}")
```

Comment:
Why do you have this here, wouldn't it be better to know why it failed when calling directly from `main`?
if you think it better, then we need a console.log/error the error